src/devices/frameGrabber_nwc_yarp/FrameGrabber_nwc_yarp.cpp

```diff
@@ -220,6 +220,11 @@ bool FrameGrabber_nwc_yarp::open(yarp::os::Searchable& config)
         yCInfo(FRAMEGRABBER_NWC_YARP) << "No remote specified. Waiting for connection";
     }
 
+    FrameGrabberOf_ForwarderWithStream<yarp::sig::ImageOf<yarp::sig::PixelRgb>>::setStreamReceiver(&streamReceiver);
```

Comment:
This block of code could be better placed at line ~202-203 after the streamReceiver.open()